core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -92,12 +93,20 @@ class AdminApiServiceSpec extends ApiServiceSpec {
       ActiveSubmission(constantData.workspace.namespace, constantData.workspace.name, constantData.submission1),
       ActiveSubmission(constantData.workspace.namespace, constantData.workspace.name, constantData.submission2))
 
-    Get(s"/admin/submissions") ~>
-      sealRoute(services.adminRoutes) ~>
-      check {
-        assertResult(StatusCodes.OK) { status }
-        assertSameElements(expected, responseAs[Seq[ActiveSubmission]])
-      }
+    val uri: Uri = "/admin/submissions"
+
+    withStatsD {
+      Get(uri) ~>
+        sealRoute(instrumentRequest { services.adminRoutes }) ~>
+        check {
+          assertResult(StatusCodes.OK) {
+            status
+          }
+          assertSameElements(expected, responseAs[Seq[ActiveSubmission]])
+        }
+    } { capturedMetrics =>
+      capturedMetrics should contain (expectedHttpRequestCounterMetric(HttpMethods.GET, uri, 1))
```

Comment:
Looks reasonable - though I'm wondering about just passing Strings directly like
```
expectedHttpRequestCounterMetric("get.admin.submissions.request", 1)
```
to make it a bit more clear the metric format we're verifying.